keel/Core/notification_backend.py

```diff
@@ -4,10 +4,16 @@
 
 
 class ConsoleEmailBackend(object):
-    def send_email(self, subject, message, sender, to=[]):
-        send_mail(subject, message, sender, to, fail_silently=False, connection=Console())
+    def send_email(self, subject, message, sender, to=[], content_type = 'html'):
+        if content_type == 'html':
+            send_mail(subject, message, sender, to, fail_silently=False, connection=Console(), html_message = message)
+        else:
+            send_mail(subject, message, sender, to, fail_silently=False, connection=Console())
 
 
 class SMTPEmailBackend(object):
-    def send_email(self, subject, message, sender, to):
-        send_mail(subject, message, sender, to, fail_silently=True)
\ No newline at end of file
+    def send_email(self, subject, message, sender, to, content_type = 'html'):
+        if content_type == 'html':
+            send_mail(subject, message, sender, to, fail_silently=False, connection=SMTP(), html_message = message)
+        else:
+            send_mail(subject, message, sender, to, fail_silently=False, connection=SMTP())
```

Comment:
I tried run in local as  Production ENV, this code was suffering from 
```TypeError: SMTPEmailBackend() takes no arguments```

When I added `connection=SMTP() ` it worked.